README.md

```diff
@@ -1,17 +1,55 @@
 # rs-disc
 
-### Database Information
-sqlite3           : 2.6.0
+## Database Information
+sqlite3           : 2.6.0  
 database filename : rs-forum.db
 
-#### Table Schemas
-Table Name : threads
-
+### Table Schemas
+#### threads
 | Column Name       |   Data Type  |
 |:-----------------:|-------------:|
+| thread_id         | integer      |
 | url               | text         |
-| last_page_integer | integer      |
-| last_body         | text         |
-| last_user         | text         |
-| last_timestamp    | text         |
+| last_page_num     | integer      |
+| last_post_num     | text         |
 | active            | integer      |
+CREATE TABLE threads (thread_id INTEGER PRIMARY KEY, url TEXT, last_page_num INTEGER, last_post_num INTEGER, active INTEGER, CONSTRAINT url UNIQUE(url));

```

Comment:
Stupid github markdown is putting these create statements in the table. Should be an easy fix